### PR TITLE
maven + windows

### PR DESCRIPTION
**Pull request by guptakuldeep**

### CODE REVIEW

File: bin/kafka-run-class.bat [hunk 1]
```diff
@@ -0,0 +1,77 @@
+@echo off
```
Comment (tnachen) [inline bin/kafka-run-class.bat:1]: Isn't there a windows folder already with scripts like these?

Comment (tnachen) [inline bin/kafka-run-class.bat:1]: Nvm, looks like they're all removed

**Comment by gwenshap** [comment]
You are trying to merge into a repository of a Kafka version that is around 4 years old and will never see another release. Perhaps close and check if we still need this improvement on trunk?

**Comment by gwenshap** [comment]
ah, it really is that old. 
I'll add it to my "ping infra to close" collection

**Pull request closed without merging**